docs/telemetry.md

```diff
@@ -50,4 +83,309 @@ Starting with Percona XtraDB Cluster 8.0.34-26-1, telemetry is enabled by defaul
+
+The DB component does NOT collect the following:
+
+* Database names
+
+* User names or credentials
+
+* Data entered by users
+
+## Locations of metrics files and telemetry history
+
+Percona stores the Metrics file in one of the following directories on the local file system. The location depends on the product.
+
+* Telemetry root path - `/usr/local/percona/telemetry`
+
+* PSMDB (mongod) root path -   `${telemetry root path}/psmdb/`
+
+* PSMDB (mongos) root path - `${telemetry root path}/psmdbs/`
+
+* PS root path -   `${telemetry root path}/ps/`
+
+* PXC root path - `${telemetry root path}/pxc/`
+
+* PG root path - `${telemetry root path}/pg/`
+
+Percona archives the telemetry history in `${telemetry root path}/history/`.
+
+### Metrics file format
+
+The Metrics file uses the Javascript Object Notation (JSON) format. Percona reserves the right to extend the current set of JSON structure attributes in the future.  
+
+The following is an example of the Metrics file content:
+
+```JSON
+{
+  "db_instance_id": "e83c568c-e140-11ee-8320-7e207666b18a",
+  "pillar_version": "8.0.35-27",
+  "active_plugins": [
+    "binlog",
+    "mysql_native_password",
+    "sha256_password",
+    "caching_sha2_password",
+    "sha2_cache_cleaner",
+    "daemon_keyring_proxy_plugin",
+    "PERFORMANCE_SCHEMA",
+    "CSV",
+    "MEMORY",
+    "InnoDB",
+    "INNODB_TRX",
+    "INNODB_CMP",
+    "INNODB_CMP_RESET",
+    "INNODB_CMPMEM",
+    "INNODB_CMPMEM_RESET",
+    "INNODB_CMP_PER_INDEX",
+    "INNODB_CMP_PER_INDEX_RESET",
+    "INNODB_BUFFER_PAGE",
+    "INNODB_BUFFER_PAGE_LRU",
+    "INNODB_BUFFER_POOL_STATS",
+    "INNODB_TEMP_TABLE_INFO",
+    "INNODB_METRICS",
+    "INNODB_FT_DEFAULT_STOPWORD",
+    "INNODB_FT_DELETED",
+    "INNODB_FT_BEING_DELETED",
+    "INNODB_FT_CONFIG",
+    "INNODB_FT_INDEX_CACHE",
+    "INNODB_FT_INDEX_TABLE",
+    "INNODB_TABLES",
+    "INNODB_TABLESTATS",
+    "INNODB_INDEXES",
+    "INNODB_TABLESPACES",
+    "INNODB_COLUMNS",
+    "INNODB_VIRTUAL",
+    "INNODB_CACHED_INDEXES",
+    "INNODB_SESSION_TEMP_TABLESPACES",
+    "MyISAM",
+    "MRG_MYISAM",
+    "TempTable",
+    "ARCHIVE",
+    "BLACKHOLE",
+    "ngram",
+    "mysqlx_cache_cleaner",
+    "mysqlx",
+    "ROCKSDB",
+    "rpl_semi_sync_source",
+    "ROCKSDB_CFSTATS",
+    "ROCKSDB_DBSTATS",
+    "ROCKSDB_PERF_CONTEXT",
+    "ROCKSDB_PERF_CONTEXT_GLOBAL",
+    "ROCKSDB_CF_OPTIONS",
+    "ROCKSDB_GLOBAL_INFO",
+    "ROCKSDB_COMPACTION_HISTORY",
+    "ROCKSDB_COMPACTION_STATS",
+    "ROCKSDB_ACTIVE_COMPACTION_STATS",
+    "ROCKSDB_DDL",
+    "ROCKSDB_INDEX_FILE_MAP",
+    "ROCKSDB_LOCKS",
+    "ROCKSDB_TRX",
+    "ROCKSDB_DEADLOCK"
+  ],
+  "active_components": [
+    "file://component_percona_telemetry"
+  ],
+  "uptime": "6185",
+  "databases_count": "7",
+  "databases_size": "33149",
+  "se_engines_in_use": [
+    "InnoDB",
+    "ROCKSDB"
+  ],
+  "replication_info": {
+    "is_semisync_source": "1",
+    "is_replica": "1"
+  }
+}
+```
+
```

Comment:
What about adding this code example as a collapsible tab? It takes up too much space on the page.